charts/cert-manager/Chart.yaml

```diff
@@ -28,3 +28,8 @@ sources:
 maintainers:
 - name: The Kubermatic Kubernetes Platform contributors
   email: support@kubermatic.com
+
+dependencies:
+- repository: https://charts.jetstack.io
+  name: cert-manager
+  version: 1.5.2
```

Comment:
Your editor is not adding trailing newlines to the end of files. 😁

Reply:
It will now ;-) also fixed in the code.